client/src/components/Allocations/AllocationsCells.js

```diff
@@ -25,17 +29,41 @@ export const Team = ({ cell: { value } }) => {
             type: 'GET_TEAMS',
             payload: { ...value }
           });
-          setOpenModal(true);
+          setTeamModal(true);
         }}
-        disabled={openModal}
+        disabled={teamModal}
       >
         View Team
       </Button>
       <AllocationsTeamViewModal
-        isOpen={openModal}
-        pid={projectId}
-        toggle={() => setOpenModal(!openModal)}
+        isOpen={teamModal}
+        projectId={projectId}
+        toggle={() => setTeamModal(!teamModal)}
       />
+      {isPi && (
```

Comment:
we need to support delegate as well.  delegate information is not provided on project listing (does TAS have a todo on this?); we don't know who the delegate is until we query the actual project.  so @rstijerina 's idea was to have an edit button in the AllocationsTeamView modal.

Reply:
Yes, @owaisj is going to do this in https://jira.tacc.utexas.edu/browse/FP-1269

Reply:
👍